mlflow/gateway/app.py

```diff
@@ -206,9 +209,16 @@ async def search_routes(page_token: Optional[str] = None) -> SearchRoutesRespons
             result["next_page_token"] = next_page_token.encode()
 
         return result
+    
+    @app.get(MLFLOW_GATEWAY_LIMITS_BASE + "{route}")
+    async def get_limits(route: str) -> LimitsConfig:
```

Comment:
Does fastapi complain if we remove `route`?

Reply:
Yes, we need to keep `route` as param: https://fastapi.tiangolo.com/tutorial/path-params/